tests/wpt/mozilla/meta/mozilla/webgl/context_creation_error.html.ini

```diff
@@ -1,5 +1,5 @@
 [context_creation_error.html]
   type: reftest
+  prefs: ["webgl.testing.context_creation_error:true"]
   [WebGLContextEvent "webglcontextcreationerror" event]
```

Comment:
We should remove this and the following line, right? The test should pass now :)